core/src/integration-test/java/com/scalar/db/api/DistributedTransactionAdminIntegrationTestBase.java

```diff
@@ -67,7 +67,7 @@ public abstract class DistributedTransactionAdminIntegrationTestBase {
           .addSecondaryIndex(COL_NAME5)
           .addSecondaryIndex(COL_NAME6)
           .build();
-
+  private TransactionFactory transactionFactory;
   private DistributedTransactionAdmin admin;
   private DistributedTransactionManager manager;
```

Comment:
Ditto. Can you please also get a manager instance in `truncateTable_ShouldTruncateProperly()` similar to the test methods you added/modified in this PR? So we can remove this `manager ` field.